workers/data_refinery_workers/downloaders/management/commands/queue_task.py

```diff
@@ -0,0 +1,47 @@
+from django.core.management.base import BaseCommand
+from data_refinery_models.models import (
+    SurveyJob,
+    Batch,
+    BatchStatuses,
+    DownloaderJob
+)
+from data_refinery_workers.downloaders.array_express \
+    import download_array_express
+
+
+# Import and set logger
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+# Just a temporary way to queue a celery task
+# without running the surveyor.
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        # Create all the dummy data that would have been created
+        # before a downloader job could have been generated.
+        survey_job = SurveyJob(
+            source_type="ARRAY_EXPRESS"
+        )
+        survey_job.save()
+
+        batch = Batch(
+            survey_job=survey_job,
+            source_type="ARRAY_EXPRESS",
+            size_in_bytes=0,
+            download_url="http://www.ebi.ac.uk/arrayexpress/files/E-MTAB-3050/E-MTAB-3050.raw.1.zip",  # noqa
+            raw_format="MICRO_ARRAY",
+            processed_format="PCL",
+            pipeline_required="MICRO_ARRAY_TO_PCL",
+            accession_code="A-AFFY-1",
+            internal_location="expression_data/array_express/A-AFFY-1/",
```

Comment:
I meant lines 18 and 19 to clarify that, but I agree it can be more clear. I'll change the name to queue_test_task.py. But yeah it's only for testing, not to be used in production ever.